src/app/screens/Home/DefaultView/index.tsx

```diff
@@ -53,6 +57,18 @@ const DefaultView: FC<Props> = (props) => {
 
   const itemsLimit = 8;
 
+  async function getLightningAddress() {
+    setLoadingLightningAddress(true);
+    const response = await api.getAccountInfo();
+    const lightningAddress = response.info.lightning_address;
+    if (lightningAddress) setLightningAddress(lightningAddress);
+    setLoadingLightningAddress(false);
+  }
+
+  useEffect(() => {
+    getLightningAddress();
```

Comment:
should this be a hook? `useLightningAddress()`?

Reply:
Yes, good idea! :+1: